conf/machine/raspberrypi0-2w-64.conf

```diff
@@ -0,0 +1,21 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi0 2 Wifi Development Board
+#@DESCRIPTION: Machine configuration for the RaspberryPi0 2 Wifi in 64 bits mode
+
+MACHINEOVERRIDES = "raspberrypi3-64:${MACHINE}"
+
+include conf/machine/raspberrypi3-64.conf
+
+MACHINE_EXTRA_RRECOMMENDS += "\
+    linux-firmware-rpidistro-bcm43436 \
+    linux-firmware-rpidistro-bcm43436s \
+    bluez-firmware-rpidistro-bcm43430b0-hcd \
+"
+
+RPI_KERNEL_DEVICETREE = " \
+    broadcom/bcm2710-rpi-zero-2.dtb \
+    "
+
+# This is necessary because the Pi Zero 2 Wifi 64bit overrides the Pi3 64bit,
```

Comment:
Isn't this because the include overwrites your `MACHINEOVERRIDES`?

Reply:
Thanks @agherzan, indeed, the include does replace it and this is how it looks like without the last assignment on it:

MACHINEOVERRIDES="rpi:aarch64:armv8a:raspberrypi3:raspberrypi0-2w-64"

If we use `MACHINEOVERRIDES="raspberrypi3-64:${MACHINE}"` after the include then recipes
will complain about raspberrypi0-2w-64 not present in COMPATIBLE_MACHINE, since rpi is not in MACHINEOVERRIDES.

I modified the PR to do the replace after the include as you suggested, alternatively we could hardcode MACHINEOVERRIDES:
```
MACHINEOVERRIDES = "rpi:armv7ve:raspberrypi3:${MACHINE}"
MACHINEOVERRIDES = "rpi:aarch64:armv8a:raspberrypi3:raspberrypi3-64:${MACHINE}"
```

Reply:
hardcoding is not a good idea.

Reply:
Sure @kraj will leave the overrides as they are, if it's all good should we merge this?

Reply:
yes